V2/tests/api/azure/time_entry_azure_endpoints_test.py

```diff
@@ -76,3 +77,59 @@ def test__delete_time_entries_azure_endpoint__returns_a_status_code_400__when_ti
 
     assert response.status_code == 400
     assert response.get_body() == b'Invalid Format ID'
+
+
+def test__update_time_entry_azure_endpoint__returns_an_time_entry__when_found_an_time_entry_to_update(
+    test_db, time_entry_factory, insert_time_entry, activity_factory, insert_activity
+):
+    inserted_activity = insert_activity(activity_factory(), test_db)
+    existent_time_entries = time_entry_factory(activity_id=inserted_activity.id, technologies="[jira,sql]")
```

Comment:
Here is a code smell from SonarCloud, if the fixture time_entry_factory needs to be sent the same parameter every time maybe we could change the fixture if nothing else is affected